lib/proxy/log.rb

```diff
@@ -20,6 +20,46 @@ def logger
   end
 
   class LoggerFactory
+    class LogFormatter
+      Format = "%s, [%s%s] %5s -- %s: %s\n".freeze
+
+      def call(severity, time, progname, msg)
+        Format % [severity, format_datetime(time), request_id[0..5], severity, progname, msg2str(msg)]
```

Comment:
The first `severity` field used to be a single letter (`severity[0..0]` in stdlib) so it's now duplicated and affects the indentation. Previously:

<pre>
D, [2016-11-10T13:34:04.946305 #20953] DEBUG -- : require_ssl_client_verification: skipping, non-HTTPS request
I, [2016-11-10T13:34:04.946767 #20953]  INFO -- : 192.168.122.1 - - [10/Nov/2016 13:34:04] "GET /tftp/serverName HTTP/1.1" 200 30 0.0007
</pre>

Now:

<pre>
DEBUG, [2016-11-15T13:12:52.052872 ] DEBUG -- : Rack::Handler::WEBrick is mounted on /.
INFO, [2016-11-15T13:12:52.052942 ]  INFO -- : WEBrick::HTTPServer#start: pid=4326 port=8443
</pre>
</pre>